docker/pre-install

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Modified version of github.com/spdk/docker
+
+set -e
+
+spdk_repo=/tmp/spdk
+
+cleanup() {
+
+	rm -f "$HOME/rpmbuild/rpm/x86_64/"*.rpm
+	rm -f "$spdk_tar"
+}
+
+trap 'cleanup' EXIT
+
+# Spice it a bit with supported sources
+"$spdk_repo/scripts/pkgdep.sh" -d
+
+
+# "$spdk_repo/test/common/config/vm_setup.sh" --test-conf=fio
+
+
+# HACK: In case we received a .tar with built SPDK we need to overwrite the
+# configuration to update all the paths make would need to lookup - this is
+# needed since we execute inside a different mount namespace so we won't be
+# able to find any absolute paths that were used prior creating the .tar.
+
+"$spdk_repo/configure" --with-rbd > /dev/null
+# spdk_version=$(cat $spdk_repo/spdk/VERSION)
+
+# Deploy SPDK inside the container
+echo "running rpm build, ${spdk_version}"
+SPDK_VERSION="${spdk_version}" DEPS="no" "$spdk_repo/rpmbuild/rpm.sh"
+
+mkdir -p /tmp/rpms
+echo "spdk_version is $spdk_version"
+ls -Rl "$HOME/rpmbuild"
+mv "$HOME/rpmbuild/rpm/x86_64/"*.rpm /tmp/rpms/
+# dnf clean all
+echo "dnf clean all"
```

Comment:
Same here, `dnf clean all` is echoed but isn't actually run?  Please audit all commented out commands and remove those that aren't needed along with context.

Reply:
removed the commented out lines